tests/foreman/installer/test_installer.py

```diff
@@ -195,10 +197,10 @@ def test_capsule_installation(sat_non_default_install, cap_ready_rhel, setting_u
     :id: 64fa85b6-96e6-4fea-bea4-a30539d59e65
 
     :steps:
-        1. Get a fapolicyd enabled Satellite
-        2. Configure capsule repos
-        3. Install and enable fapolicyd
-        4. Enable capsule module
+        1. Use Satellite with fapolicyd enabled (non-default)
```

Comment:
I see this uses `sat_non_default_install` but I really disagree that we should be testing with `--foreman-rails-cache-store type:file`. If we want proper integration testing we should be testing a basic default Satellite instead of the edge case.

If anything, the test that actually wants to test Satellite with non-Redis should explicitly configure Satellite that way.

Reply:
I decided to create another fixture with default setup but including fapolicyd. Let's consider this as a workaround I hope it can be done better (e.g. parametrized), but it's out of scope of this PR.